.github/workflows/ci.yml

```diff
@@ -34,14 +34,13 @@ jobs:
       - name: Standard (Lint)
         run: bundle exec rake standard
   test:
-    name: Specs
+    name: "Specs (Rack 2 and 3)"
     runs-on: ubuntu-22.04
     if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository
     strategy:
       matrix:
         ruby:
           - "2.7"
-          - "3.2"
```

Comment:
Yes, because then the matrix turns into 4 versions - and I had some trouble loading older ActiveRecord on the newer Ruby versions. I presume what works on 2.7 will work on 3.x anyway, so it felt like a good comproimse